File: MODUL6/C/PRAK603-2410817310002-MuhammadRakhaAthallah.c
#include <stdio.h>

int main() {
    int a, b;
    scanf("%d %d", &a, &b);
    if (a != b) {
        printf("Jumlah tidak sama");
        return 1;
    }
    int arr1[a], arr2[b];
    for (int i = 0; i < a; ++i) scanf("%d", &arr1[i]);
    for (int i = 0; i < b; ++i) scanf("%d", &arr2[i]);
    for (int i = 0; i < a; ++i) printf("%d ", arr1[i] * arr2[i]);
    return 0;
}

File: MODUL6/PYTHON/PRAK603-2410817310002-MuhammadRakhaAthallah.py
a, b = map(int, input().split())
if a != b:
    print("Jumlah tidak sama")
else:
    list_a = list(map(int, input().split()))
    list_b = list(map(int, input().split()))
    print(*[list_a[i] * list_b[i] for i in range(a)])